obs.py

```diff
@@ -207,6 +169,9 @@ def __init__(self, name, config):
         if not os.path.exists(g_dev["cam"].site_path + "archive"):
             os.makedirs(g_dev["cam"].site_path + "archive")
 
+        self.last_solve_time = datetime.datetime.now() - datetime.timedelta(days=1)
+        self.images_since_last_solve = 10000
```

Comment:
This keeps tracks of how many images since the last platesolve. It is set to a stupidly high value and gets reset to 1 after the first solve. This is part of the timer that only solves every X image and only are Y amount of time. Otherwise the scope bounces around the sky.